res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -1814,13 +1819,17 @@ NumarkMixtrack3.FXButton = function(channel, control, value, status, group) {
                 deck.LEDs["fx" + effectNum].flashOn(250, ON, 250);
             }
         }
-    } else if (deck.PADMode && value === DOWN) {
+    } else if (deck.shiftKey && value === DOWN) {
         // load next effect, make sure it is inactive, and make sure the unit is enabled
         engine.setValue(effectGroup, "next_effect", true);
         engine.setValue(effectGroup, "enabled", false);
         engine.setValue("[EffectRack1_EffectUnit" + decknum + "]", "group_[Channel" + decknum + "]_enable", true);
     } else if (deck.TapDown && value === DOWN) {
-        deck.focusEffect(effectNum);
+        if (deck.focusedEffect() === effectNum) {
+            deck.focusedEffect(0);
```

Comment:
I thought the same as well, I chose `focusedEffect` because when calling it without any arguments it returns the focusedEffect. It's awkward though and I'll change it next time I have time to look at the code.